### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dahuang37**
Screen Shot 2021-04-03 at 10 10 07 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+    <img src="https://admin.peacemakr.io/p_logo.png" width="150"/>
   <br>
 </p>
 
 # Peacemakr E2E-Encryption-as-a-Service Python SDK
 
 ![Build Status](https://github.com/peacemakr-io/peacemakr-python-sdk/workflows/Build%20%26%20Test/badge.svg)
 
-Peacemakr's E2E-Encryption-as-a-Service SDK solves all the hard problem of E2E-Encryption, and provides consistent solution with a simple interface.
+Peacemakr's E2E-Encryption-as-a-Service SDK simplify your data security with E2E-Encryption service and automated key lifecycle management.
```
Comment (interstellarPotato) [inline README.md:11]: simplify-> simplifies

File: README.md [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+
+You can easily encrypt your data without worrying about backward compatibility, cross platform portability, or changing security requirements.
```
Comment (interstellarPotato) [inline README.md:13]: Suggestion: "or changing security requirements." -> "or changing **ever** security requirements."

**Review by interstellarPotato — APPROVED** [review]
LGTM, just a few minor comments/suggestions :)

File: README.md (hunk 2 above)
Comment (notasecret) [reply README.md:13]: sorry that change doesn't make sense to me

File: README.md (hunk 1 above)
Comment (notasecret) [reply README.md:11]: yes

**Pull request merged**